docs/how-tos/using-pynxtools-test-framework.md

```diff
@@ -0,0 +1,56 @@
+# Generalized Test Functionality for `pynxtools` plugins
+The pynxtools sub-package testing is to be utilized to write automated tests for pynxtools reader plugins without requiring in-depth knowledge of the pynxtools internal architecture. The tool supports generalised a general test for all reader plugins, irrespective of the technical details of the raw data files and the internal design of the plugin (note: it is assumed that the plugin was built from the [plugin template](https://github.com/FAIRmat-NFDI/pynxtools-plugin-template) or has the same structure internally).
```

Comment:
Why did you remove the quotes here? I would go with 
"The `pynxtools` sub-package `testing` is to be utilized to write automated test for `pynxtools`..."

Reply:
Noted!